.github/workflows/unix.yml

```diff
@@ -181,3 +181,33 @@ jobs:
       - name: Build
         run: ninja
         working-directory: build
+
+  macos:
+    needs: buildcheck
+    strategy:
+      fail-fast: false
+      matrix:
+        os: # macos-14 runs on aarch64; the others run on x64
```

Comment:
Can we also run macos-14 tests on x86? If it is possible I think we should since apple still supports MacOS 14 on x86.

Reply:
Fair enough! Thanks for checking. :)